### PR TITLE
added opening url from xcrun simctl command

### PR DESCRIPTION
**Pull request by sergey-plevako-badoo**
`xcrun simctl openurl Booted http://example.com`

### CODE REVIEW

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/data/UrlPath.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.badoo.automation.deviceserver.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.nio.file.Path
+
+class UrlPath(
+    @JsonProperty("url")
+    val url: String)
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/data/UrlPath.kt]: add new line

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt [hunk 2]
```diff
@@ -615,6 +615,12 @@ class Simulator (
         return result.isSuccess
     }
 
+    override fun openUrl(url: String) : Boolean {
+        val command = listOf("xcrun", "simctl", "openurl", udid, url)
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt]: what will happen if url contains '&' symbol?

**Review by fr0l — CHANGES_REQUESTED** [review]

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt (hunk 2 above)
Comment (sergey-plevako-badoo) [reply device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt]: fixed by adding the quotes around

**Comment by fr0l** [comment]
@sergey-plevako-badoo ok. merging. best if we could URL validation (just a security precaution against ";drop database" and so on)

**Pull request merged**